src/cli/cluster/cluster_manager.js

```diff
@@ -181,6 +181,7 @@ export default class ClusterManager {
       ignored: [
         /[\\\/](\..*|node_modules|bower_components|public|__[a-z0-9_]+__|coverage)[\\\/]/,
         /\.test\.js$/,
+        /\/cypress\//,
```

Comment:
This keeps Kibana from starting whenever files in cypress are modified. Probably needs a tighter check but works for now